crates/weaver_forge/src/extensions/code.rs

```diff
@@ -4,7 +4,16 @@
 
 use std::collections::HashMap;
 
-use minijinja::Value;
+use crate::config::TargetConfig;
+use minijinja::{Environment, Value};
+
+/// Add code-oriented filters to the environment.
+pub(crate) fn add_filters(env: &mut Environment<'_>, config: &TargetConfig) {
+    env.add_filter("type_mapping", type_mapping(config.type_mapping.clone()));
+    env.add_filter("map_text", map_text(config.text_maps.clone()));
+    env.add_filter("comment_with_prefix", comment_with_prefix);
```

Comment:
FYI - we may want to have a `javadoc_comment` or `wrapped_comment` option (for `/** */` and `/* */` style comments)

Reply:
Can't we use the existing `comment_with_prefix` I used for the Rust experiment?

Reply:
Something like:

```
/**
{{ attribute.brief | comment_with_prefix(" * ") }}
**/
```

Reply:
Didn't think of that, yes that would work.